### ISSUE TITLE
Can’t reproduce your results in your cvpr paper

### ISSUE THREAD

**Issue by yflv-yanxia** [summerlvsong/Aggregation-Cross-Entropy#17]
I reproduced crnn+ctc and test it on IIIT5K+SVT+IC03+IC13 test database, got WER 0.153, which is same as the reported results in paper. 
I also reproduced crnn+ace loss, but only got WER 0.205 on the same test database, any advise?
My environment:
pytorch 1.2.0
batchsize 60
trained only on 8-million synthetic data released by Jaderberg
iterations 1000k
adadelta rho 0.9

**Comment by MiTeng0215** [comment]
Hi, can you share the code of crnn + ace in the 1D prediction, I also reproduced it on the IC03 data,.But,my accuracy rate has always been 0, I think I have modified the wrong code. Can i refer to your code ?my email is miteng0215@163.com,thank you.

**Comment by MiTeng0215** [comment]
> I reproduced crnn+ctc and test it on IIIT5K+SVT+IC03+IC13 test database, got WER 0.153, which is same as the reported results in paper.
> I also reproduced crnn+ace loss, but only got WER 0.205 on the same test database, any advise?
> My environment:
> pytorch 1.2.0
> batchsize 60
> trained only on 8-million synthetic data released by Jaderberg
> iterations 1000k
> adadelta rho 0.9

Hi, can you share the code of crnn + ace in the 1D prediction, I also reproduced it on the IC03 data,.But,my accuracy rate has always been 0, I think I have modified the wrong code. Can i refer to your code ?my email is miteng0215@163.com,thank you.

**Comment by MiTeng0215** [comment]
> I reproduced crnn+ctc and test it on IIIT5K+SVT+IC03+IC13 test database, got WER 0.153, which is same as the reported results in paper.
> I also reproduced crnn+ace loss, but only got WER 0.205 on the same test database, any advise?
> My environment:
> pytorch 1.2.0
> batchsize 60
> trained only on 8-million synthetic data released by Jaderberg
> iterations 1000k
> adadelta rho 0.9

I have the same results as you, and I want to know if you have any new progress.

**Comment by bjlgcxc** [comment]
it's hard for me to train crnn+ace loss to coverage, do you know why?

**Comment by zhangfengyo** [comment]
> I reproduced crnn+ctc and test it on IIIT5K+SVT+IC03+IC13 test database, got WER 0.153, which is same as the reported results in paper.
> I also reproduced crnn+ace loss, but only got WER 0.205 on the same test database, any advise?
> My environment:
> pytorch 1.2.0
> batchsize 60
> trained only on 8-million synthetic data released by Jaderberg
> iterations 1000k
> adadelta rho 0.9

maybe some augmentation are used without mention in the paper
